docs/track-click.md

```diff
@@ -0,0 +1,60 @@
+# Track click script
+
+The gem includes a script that allows tracking to be added to clicked elements with relative ease. It depends upon the main analytics code to function.
+
+Basic use:
+
+```html
+<a href="/link" data-module="gem-track-click" data-track-category="category" data-track-action="action" data-track-label="Foo">Link</a>
+```
+
+Advanced use. Adds a custom dimension of `dimension29` with a value of `dimension-value`, and a `value` of `9`.
+
+```html
+<a href="/link" data-module="gem-track-click" data-track-category="category" data-track-action="action" data-track-label="label" data-track-dimension="dimension-value" data-track-dimension-index="29" data-track-value="9">Link</a>
+```
+
+Track with arbitrary JSON:
+
+```html
+<a href='/link' data-module='gem-track-click' data-track-category='category' data-track-action='1' data-track-label='/' data-track-options='{\"dimension28\": \"foo\", \"dimension29\": \"bar\"}'>Link</a>
```

Comment:
I don't think you need to escape the double quotes here as `JSON.parse()` doesn't need it and it's inside single quotes:

```suggestion
<a href='/link' data-module='gem-track-click' data-track-category='category' data-track-action='1' data-track-label='/' data-track-options='{"dimension28": "foo", "dimension29": "bar"}'>Link</a>
```

Reply:
Of course, thanks for spotting! Have corrected. I've also reformatted all the examples so they're not on a single line, made it a bit more readable.